pwa/components/common/listItem/domainBlock.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from "react";
+import {Block} from "./block";
+import {Icon} from "../icon";
+import {Subdomain} from "./subdomain";
+import {EditBlock} from "./editBlock";
+import {Popup} from "../popup";
+
+type domainBlockProps = {
+  number: number,
+  name: string,
+  countConfiguration: number,
+}
+export const DomainBlock: React.FC<domainBlockProps> = ({number,name, countConfiguration}) => {
+  const [open, setOpen] = useState(false);
+  const [showPopup, setShowPopup] = useState(false);
+
+  const handleClickPopup = () => {
+    setShowPopup(true);
+    setTimeout(() => {
+      setShowPopup(false);
+    }, 3000);
+  };
+
+  return (
+    <>
+      {showPopup && (
+        <div>
```

Comment:
Are these divs necessary?

Reply:
Yes because it places the popup on the domain where we modify a subdomain. If we remove this div, the popup will appear at the top of the parent and if we have a lot of domain or a small device screen we wont see it.

Reply:
With the div: 
![image](https://user-images.githubusercontent.com/78682200/236170106-3131cfd2-1c6f-4999-a4d1-8c42780bec02.png)

Without: 
![image](https://user-images.githubusercontent.com/78682200/236170327-0ae1aa8a-9066-4293-bb8d-a668f0fcb96c.png)